docs/aurora/containers/containers.md

```diff
@@ -0,0 +1,109 @@
+# Containers on Aurora
+Aurora, powered by 2 Intel Xeon CPUs and 6 Intel Data Center GPU Max Series, benefits from container-based workloads for seamless compatibility across Intel systems. This guide details the use of containers on Aurora, including custom container creation, large-scale execution, and common pitfalls.
+
+We support Apptainer and Podman (documentation coming soon) on Aurora.
+
+## Apptainer
+
+### Setup
+
+Aurora employs Apptainer (formerly known as Singularity) for container management. To set up Apptainer, run on a compute node the following:
```

Comment:
```suggestion
Aurora employs Apptainer (formerly known as Singularity) for container management. To set up Apptainer, run the following on a compute node:
```
Maybe cite the Running Jobs page, like at the top of https://docs.alcf.anl.gov/aurora/data-science/frameworks/gpytorch/ ?